### PR TITLE
Finished week 2 portfolio 

### PR DESCRIPTION
**Pull request by roroross**
Finished MVP of week 2 portfolio

### CODE REVIEW

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
       <h1>Stanley 🐈</h1>
       <p>This is Stanley.</p>
       <img src="images/stanley-1.jpg"/>
-
+
```
Comment (Techget) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:13]: nit unnecessary space

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html [hunk 2]
```diff
@@ -22,6 +22,7 @@ <h1>Stanley 🐈</h1>
 
       <p>Click <a href="images.html">here</a> to see more pictures of Stanley!</p>
       <p>Click <a href="random.html">here</a> to see a random picture of Stanley!</p>
+
```
Comment (Techget) [inline walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:25]: nit unnecessary new line

File: walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html (hunk 1 above)
Comment (roroross) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:13]: hahah This was actually the Google walkthrough code! :')

Comment (Techget) [reply walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html:13]: Yea, so i don't expect any change to this file.

**Review by Techget — APPROVED** [review]
Overall looks good, I will use more formal review workflow next time, sorry for making you feel confused.

**Pull request merged**